scripts/systemd/serial-keel.template.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Serial Keel Server
+
+[Service]
+User=[user-name-here]
+WorkingDirectory=/home/[user-name-here]/serial-keel/
```

Comment:
Should this matter?
Since we do `cargo install` the binary should be located in this user's PATH.

Reply:
Removed this WorkingDirectory line. However we needed to cd into the repo directory on service start to do the `git rev-parse`, which is now being done in the script itself.